notification.go

```diff
@@ -171,6 +176,56 @@ func (n *notifier) GetServerInformation() (ServerInformation, error) {
 	return ret, nil
 }
 
+type NotificationClosedSignal struct {
+	Id uint32
+	Reason uint32
+}
+
+func (n *notifier) NotificationClosed(closed chan<- *NotificationClosedSignal) {
+	c := make(chan *dbus.Signal, len(closed))
+
+	go (func() {
+		for {
```

Comment:
Yes, I could make the function return a channel. When the channel is closed, call `n.conn.Signal(c)` another time to remove it and stop the routine. What do you think?